client/components/date-range/README.md

```diff
@@ -24,6 +24,7 @@ Props are displayed as a table with Name, Type, Default, and Description as head
 
 **Required props are marked with `*`.**
 
+<<<<<<< HEAD
```

Comment:
We should probably resolve the conflicts here.

Reply:
Oh yeah totally. Sorry about that, looks like my rebase went a bit sideways 🙈